content/docs/atom-guide/look-dev/materials/create-standardpbr-material.md

```diff
@@ -13,30 +13,30 @@ This tutorial covers the following concepts:
 - Using texture file masks
 
 ## Before starting
-A material can have multiple textures, such as base color, normal map, and roughness. These textures contain different data that are combined to create the overall look of the material. You can design the textures for a material using external tools, such as [Substance Designer](https://www.substance3d.com/products/substance-designer/) and [Materialize](http://boundingboxsoftware.com/materialize/). In this tutorial, we use ready-made materials from [Texture Haven](https://texturehaven.com/). 
+A material can have multiple textures, such as base color, normal map, and roughness. These textures contain different data that are combined to create the overall look of the material. You can design the textures for a material using external tools, such as [Substance Designer](https://www.adobe.com/products/substance3d-designer.html) and [Materialize](http://boundingboxsoftware.com/materialize/). In this tutorial, we use ready-made materials from [Poly Haven](https://polyhaven.com/). 
 
 ### Download texture files
-Download the [castle brick material](https://texturehaven.com/tex/?t=castle_brick_02_red) in any resolution from Texture Haven. When downloading, make sure to choose **All Maps** to download all textures that make up this material.
+Download the [castle brick material](https://polyhaven.com/a/castle_brick_02_red) in any resolution from Poly Haven. To download all the required maps, click the hamburger button next to the **Download** button, then click buttons with any of the file extensions next to AO, Diffuse, Displacement, Normal, and Rough.
```

Comment:
```suggestion
Download the [castle brick material](https://polyhaven.com/a/castle_brick_02_red) in any resolution from Poly Haven. To download the required maps, click the hamburger button next to the **Download** button, then choose an extension to use and click the button for that extension next to the AO, Diffuse, Displacement, Normal, and Rough maps.  Save the files to disk if they do not download automatically.
```
Very minor suggestion: remove 'all', it would make more sense if the next step downloaded every map that is required. 

I added an additional step to choose (in your head) a specific extension.  I'm hoping that makes the rest of this step clearer.  Also I noticed that .exr downloads automatically, but .jpg and .png files displayed in-browser without downloading (requiring the user to right-click save).  I wish they had kept the download all button!

Reply:
The changes suggested make the instruction clearer, the .jpg/.png download method did not occur to me at the time. Thanks for submitting this along with explanation!